scripts/create-component/plop-templates-component/src/components/{{componentName}}/{{componentName}}.test.tsx.hbs

```diff
@@ -10,21 +9,10 @@ describe('{{componentName}}', () => {
     displayName: '{{componentName}}',
   });
 
-  let wrapper: ReactWrapper | undefined;
+  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
 
-  afterEach(() => {
-    if (wrapper) {
-      wrapper.unmount();
-      wrapper = undefined;
-    }
-  });
-
-  /**
-   * Note: see more visual regression tests for {{componentName}} in /apps/vr-tests.
-   */
   it('renders a default state', () => {
-    const component = renderer.create(<{{componentName}}>Default {{componentName}}</{{componentName}}>);
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    const result = render(<{{componentName}}>Default {{componentName}}</{{componentName}}>);
+    expect(result.container).toMatchSnapshot();
```

Comment:
**suggestion:** can we remove using snapshots for component content for our boilerplate? in general using snapshots for react component testing is a well known antipattern, thus I don't think it's good approach to encourage folks using it.

Reply:
I think removing snapshot tests from our components is a worthwhile topic for an RFC; would you mind writing that up? Given that the existing template already used snapshots, I'm going to leave it as-is for this PR. But it can be removed in the future if people agree with the RFC.